<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenUri</class>
 <widget class="QDialog" name="OpenUri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open URI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resource.qrc">
    <normaloff>:/ico/icon.ico</normaloff>:/ico/icon.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_lDescription">
     <property name="text">
      <string>Open payment request from URI or file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_lUri">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>URI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_leUri">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnBrowse">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_bthCancel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_btnOk</sender>
   <signal>clicked()</signal>
   <receiver>OpenUri</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_bthCancel</sender>
   <signal>clicked()</signal>
   <receiver>OpenUri</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
